### ISSUE TITLE
Undefined variable: name (With  Closure Rule)

### ISSUE THREAD

**Issue by ocordeiro** [kristijanhusak/laravel-form-builder#427]
I have updated to v 1.14.0 and received this error on abstract class FormField line 674

![captura de tela 2018-05-10 as 18 39 05](https://user-images.githubusercontent.com/1471463/39899440-76e3a360-5481-11e8-8974-956b25f300c2.png)

**Comment by alamcordeiro** [comment]
https://github.com/kristijanhusak/laravel-form-builder/blob/8d1c67102e20c485717823f1be02b4b2235d072f/src/Kris/LaravelFormBuilder/Fields/FormField.php#L674

**Comment by ocordeiro** [comment]
Solution:
just add `` use($name)``

```        if (is_array($rules)) {
            $rules = array_map(function($rule) use($name) {
                if ($rule instanceof \Closure) {
                    return $rule($name);
                }
                return $rule;
            }, $rules);
        }
```

**Comment by kristijanhusak** [comment]
This is fixed in the latest 1.15.1 version.
